### ISSUE TITLE
FR: Query threads of notes

### ISSUE THREAD

**Issue by crazymedguy** [SkepticMystic/breadcrumbs#339]
In essence, a "thread" is a sequence of thought that should be treated as a "body" of knowledge. For example, when we search for a keyword in a forum, we get search results to an entire thread, pin-pointing the exact location (sequence) of the post within that forum thread. Likewise, if we retweet a twitter post, we can also retrieve the entire forum thread.

In a conversation with Robert Halsfield, he made a really good point that these threads of thought should be query-able as an entire body. It's easy enough to pull up the search and find the particular post, however, displaying it within a query on a synthesis page, for example, is a different issue. Wondering if it's possible to query for a keyword which then displays a result for the thread containing that note? Ideally it should be able to show the entire thread while pinpointing the exact location of the note.

**Comment by classicrob** [comment]
Yup. Idk I generally look at "page atomicity" as "contains all of the words I might want to search together." If Breadcrumbs is just a navigational tool, that doesn't change too much for my concept of page atomicity. However, if I can graph query threads and trees, that would unlock a ton in Breadcrumbs for me.

**Comment by classicrob** [comment]
This thread in the forum goes into detail about how the Roam and Obsidian paradigms are different, and gets at how blocks and indentation work to chain together units of information in Roam. My impression is that a lot of people use Breadcrumbs as an alternative to that sort of indentation.

**Comment by AMGMNPLK** [comment]
I don't know if we are asking the same. I would like to make queries within a Breadcrumbs notes branch. Is this possible right now?

And, perhaps, add other search criteria such as go up, siblings...

**Comment by SkepticMystic** [comment]
Could you give a specific example? I kind of get the "theory" of the request, but can't see how it would work in practice
